src/utils/validation.ts

```diff
@@ -31,3 +39,46 @@ export const isTestnet = (): boolean => {
 
   return network === 'mainnet' ? false : true;
 };
+
+export const getOnchainMetadataVersion = (onchainMetadata: Asset['onchain_metadata']): 1 | 2 => {
+  if (!onchainMetadata?.version) {
+    return 1;
+  }
+
+  if (Number(onchainMetadata.version) === 2) {
+    return 2;
+  }
+
+  return 1;
```

Comment:
just a random thought, if cardano introduces v3 then we will treat it as v1, not sure if that's a good idea but idk really. just something to keep in mind.